### DISCUSSION TITLE
Masking frequencies with phantom carriers (26Mhz multiples)

### DISCUSSION THREAD

**Discussion by iu0azb** [egzumer/uv-k5-firmware-custom#323] [Ideas]
I would suggest to mask/lockout all harmonic frequencies of 26Mhz, to prevent these from affecting scanning and spectrum operations.
This problem is usually fixed on Uniden receiver locking out these.. in some Yaesu receiver this can't be done but you can shift a bit the reference to allow normal reception on that bunch of frequencies.. that can't be appliable here on K5.

**Comment by ErikS-web** [comment]
Did you read #239. Result: `Shielding` all area over `X-tal and BK4819` seems to help.

and ...

When I use an `external antenne` on my roof, there's no problem... no carrier (opening de squelch, set on level 1) on 26, 52, 78, 104, etc.

**Reply by iu0azb** [reply]
Hi Erik,
Let give a try with the original antenna.. with external indeed the problem doesn't appear.. but using as portable you should face it. 
I've already checked on my two K5 and confirmed from two local Hamradio.

#239 is mine ;)

**Comment by gartnl** [comment]
I get your point. But 78 and 104 are legit broadcast FM frequencies. 52 is in the 6-meter band. The harmonic is very low on my UV-5R, S1 or less. Upping the squelch is enough.
In the spectrum mode it is allready possible to lockout a frequencie. So it is only a problem when scanning between VFO's.

**Reply by iu0azb** [reply]
In my radios 130mhz are having quite high level 78too 
I'll provide a detailed data as soon I can ;)
